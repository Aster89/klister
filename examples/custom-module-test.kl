#lang "custom-module.kl"

(example "inside module")
