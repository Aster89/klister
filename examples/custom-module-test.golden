"before module" : String
"inside module" : String
"after module" : String
